win32_event_log/datadog_checks/win32_event_log/win32_event_log.py

```diff
@@ -53,8 +53,15 @@ def check(self, instance):
         source_names = instance.get('source_name', [])
         log_files = instance.get('log_file', [])
         event_ids = instance.get('event_id', [])
-        message_filters = instance.get('message_filters', [])
         event_format = instance.get('event_format')
+        message_filters = instance.get('message_filters', [])
+        submit_all_events = is_affirmative(instance.get('submit_all_events', self._submit_all_events))
+
+        if not submit_all_events and not (source_names or event_ids or message_filters or log_files or user or ltypes):
```

Comment:
:brainfreeze:
Let's summarize:
- if submit_all_events is True or unset and we have filters, then we do like before => 🆗 
- if submit_all_events is True or unset and we don't have filters, then we do like before => 🆗 
- if submit_all_events is False and we have filters, then we will use filters => 🆗 
- if submit_all_events is False and we don't have filters, then we will raise an exception => 🆗 
:braindefreezed: Seems good 👍